server/server.go

```diff
@@ -124,6 +124,13 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 	var bitbucketCloudClient *bitbucketcloud.Client
 	var bitbucketServerClient *bitbucketserver.Client
 	var azuredevopsClient *vcs.AzureDevopsClient
+
+	policyChecksEnabled := false
+	if userConfig.EnablePolicyChecksFlag && !(userConfig.TFEHostname != "" || userConfig.TFEToken != "") {
+		logger.Info("Policy Checks are enabled")
```

Comment:
We should probably fail fast when TFE is being used in conjunction with this flag. Otherwise it'll be a mystery as to why stuff is not working.